README.md

```diff
@@ -79,6 +79,8 @@ Usage of ./nginx-prometheus-exporter:
 
 ### Exported Metrics
 
+* Common metrics:
+    * `nginxexporter_build_info` - shows the exporter build information.
```

Comment:
Add `-` to be consistent with [`nginx_up` documentation](https://github.com/nginxinc/nginx-prometheus-exporter/pull/32/files#diff-04c6e90faac2675aa89e2176d2eec7d8R86)
```suggestion
    * `nginxexporter_build_info` -- shows the exporter build information.
```